packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1331,9 +1331,8 @@ class SelectionOverlay {
   /// Controls the fade-in and fade-out animations for the toolbar and handles.
   static const Duration fadeDuration = Duration(milliseconds: 150);
 
-  /// A pair of handles. If this is non-null, there are always 2, though the
-  /// second is hidden when the selection is collapsed.
-  List<OverlayEntry>? _handles;
+  /// A pair of handles. The second is hidden when the selection is collapsed.
```

Comment:
Is there a reason to remove the mention of non-null? This variable is nullable and if null, there are no handles.

Reply:
IMHO

1. The `?` in type already says it is nullable, so I remove "If this is non-null"
2. the new type ensures there are 2 items, so I remove "there are always 2"

Reply:
I think this is what the original comment meant: since `_handles` is nullable, it there are handles only if it's non-null. If null, no handles are shown.

Reply:
Anyway let me revert it